### PR TITLE
MAID-2527 refactor/snippet: changes MData entries forEach to listEntries

### PR DESCRIPTION
**Pull request by hunterlester**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: safe_web_api_playground/static/js/code-snippets/safeMutableDataEntries.js [hunk 1]
```diff
@@ -22,20 +22,16 @@ module.exports = {
...
 
-      // Look for console log output in your safe browser console, not this console
       try {
-        await window.safeMutableDataEntries.forEach(entriesHandle, (key, value) => {
-
-          console.log(key + ': ' + value.buf);
-
-        })
+        var entriesArray = await window.safeMutableDataEntries.listEntries(entriesHandle);
+	console.log(entriesArray);
```
Comment (bochaco) [inline safe_web_api_playground/static/js/code-snippets/safeMutableDataEntries.js]: Perhaps we can still do a forEach on the array and log the values as before so people understand the object we are returning for each entry?

**Review by bochaco — CHANGES_REQUESTED** [review]
@hunterlester , there are some issues with the formating in some of the lines, can you please fix that and we can then merge this?

**Comment by hunterlester** [comment]
formatting fixed

**Review by bochaco — APPROVED** [review]

**Comment by bochaco** [comment]
Let's merge this only once we have a version of the browser which actually supports the changed API.

**Comment by hunterlester** [comment]
PR is obsolete because it's for deprecated API. Closing.

**Pull request closed without merging**